third_party/versions.bzl

```diff
@@ -85,6 +85,7 @@ MAVEN_TEST_DEPENDENCY_VERSIONS = {
     "androidx.work:work-testing": "2.4.0",
     "com.github.bumptech.glide:mocks": "4.11.0",
     "com.google.truth:truth": "0.43",
+    "com.squareup.okhttp3:mockwebserver":"4.1.0",
     "com.squareup.retrofit2:retrofit-mock": "2.5.0",
```

Comment:
I think lint is failing because there's a space missing before the version (after the colon).

Reply:
Nice catch.